web/plates/settings.html

```diff
@@ -345,7 +345,7 @@
     <!-- End Right Col -->
 </div>
 <!-- Developer mode area -->
-<div ng-show="DeveloperMode" class="col-sm-12">
```

Comment:
Leave it only visible to dev mode for now.. CrewDog explained why it's probably a bad idea to have it visible to end users.